### PR TITLE
Fix various small bugs

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: src/Loader/FilesystemLoader.php [hunk 1]
```diff
@@ -170,7 +170,7 @@ public function exists($name)
         }
 
         try {
-            return false !== $this->findTemplate($name, false);
+            return false !== $this->findTemplate($name);
```
Comment (stof) [inline src/Loader/FilesystemLoader.php]: this change looks wrong to me. We have this as a hidden argument.

Comment (Tobion) [inline src/Loader/FilesystemLoader.php]: Is this intentional? There is logic explicitly for this: `$throw = \func_num_args() > 1 ? func_get_arg(1) : true;`

Comment (fabpot) [reply src/Loader/FilesystemLoader.php]: indeed :)

Comment (fabpot) [reply src/Loader/FilesystemLoader.php]: reverting

**Comment by fabpot** [comment]
green now!

**Pull request merged**